tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.Repository/Matter/MatterRepository.cs

```diff
@@ -44,25 +47,30 @@ public class MatterRepository:IMatterRepository
         private IExternalSharing extrnalSharing;
         private IUserRepository userRepositoy;
         private GeneralSettings generalSettings;
-        
+        private ContentTypesConfig contentTypesSettings;
+        private ICustomLogger customLogger;
+        private LogTables logTables;
+
         /// <summary>
         /// Constructory which will inject all the related dependencies related to matter
         /// </summary>
         /// <param name="search"></param>
-        public MatterRepository(ISearch search, 
-            IOptions<MatterSettings> matterSettings, 
-            IOptions<SearchSettings> searchSettings, 
-            IOptions<ListNames> listNames, 
-            ISPOAuthorization spoAuthorization, 
+        public MatterRepository(ISearch search,
+            IOptions<MatterSettings> matterSettings,
+            IOptions<SearchSettings> searchSettings,
+            IOptions<ListNames> listNames,
+            ISPOAuthorization spoAuthorization,
             ISPContentTypes spContentTypes,
-            IExternalSharing extrnalSharing, 
+            IExternalSharing extrnalSharing,
             IUserRepository userRepositoy,
-            ISPList spList, 
+            ISPList spList,
             IOptions<CamlQueries> camlQueries,
             IOptions<GeneralSettings> generalSettings,
-            IUsersDetails userdetails, 
+            IUsersDetails userdetails,
             IOptions<ErrorSettings> errorSettings,
-            
+            IOptions<ContentTypesConfig> contentTypesSettings,
```

Comment:
Contenttypesettings require to get the sitecolumn group like _MatterCenter to filter columns.